spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestReadProjection.java

```diff
@@ -64,11 +66,14 @@ public void testFullProjection() throws Exception {
 
     Record projected = writeAndRead("full_projection", schema, schema, record);
 
-    Assert.assertEquals(
-        "Should contain the correct id value", 34L, (long) projected.getField("id"));
+    assertThat((long) projected.getField("id"))
+        .as("Should contain the correct id value")
+        .isEqualTo(34L);
 
     int cmp =
         Comparators.charSequences().compare("test", (CharSequence) projected.getField("data"));
+
+    // TODO: update this Assert with AssertJ
```

Comment:
I don't understand why this can't use AssertJ assertion

Reply:
Because, everytime I try to add an import that is unused in the code, the IDE automatically removes the import. So if I need to keep the `import org.junit.Assert;` import statement, it needs to be used somewhere. 

So I just chose the first simple AssertJ statement and reverted it to use Assert.

Reply:
why do you need to keep `import org.junit.Assert`?

Reply:
> because you're changing this file it seems you'd also have to update `TestSparkReadProjection`, otherwise CI won't pass.
> 
> Alternatively you could reintroduce
> 
> ```
> import org.junit.Assert;
> import org.junit.Rule;
> import org.junit.Test;
> import org.junit.rules.TemporaryFolder;
> ```
> 
> but leave the AssertJ conversions in this file

Based on this comment, I thought we need to keep all those imports for clearing CI.

Reply:
sorry for the mis-communication, that's my bad. I didn't mean to mention `org.junit.Assert`. In order for the test to be a JUnit4 test it just needs `org.junit.Rule` / `org.junit.Test` / `org.junit.rules.TemporaryFolder`. The assertions being used can be AssertJ (which you already converted)

Reply:
ok, I'll add these tomorrow!